frontends/web/src/routes/account/send/send.tsx

```diff
@@ -158,7 +146,7 @@ class Send extends Component<Props, State> {
       return;
     }
 
-    this.setState({ signProgress: undefined, isConfirming: true });
```

Comment:
I just decided to reset it to `undefined` every time the dialog closes (using `prevIsConfirming`) because testing is too annoying, also the `signProgress` is irrelevant for BB02.